helix-core/src/main/java/org/apache/helix/task/TaskStateModelFactory.java

```diff
@@ -51,23 +52,17 @@
 public class TaskStateModelFactory extends StateModelFactory<TaskStateModel> {
   private static Logger LOG = LoggerFactory.getLogger(TaskStateModelFactory.class);
 
+  // Unit in minutes. Need a retry timeout to prevent zkClient from hanging infinitely.
+  private static int ZKCLIENT_OPERATION_RETRY_TIMEOUT = 5;
```

Comment:
Nit: constant declarations also include `final`. So it would be `private static final int`

Reply:
Wow, my 10pm brain is really sloppy... thanks for catching it.